docs/freqai.md

```diff
@@ -532,6 +534,15 @@ for each pair, for each backtesting window within the expanded `--timerange`.
 
 ---
 
+### Hyperopt
```

Comment:
@robcaulk I appreciate your help to improve the text!

Reply:
The wording seems good to me!